package/CreateTree.test.js

```diff
@@ -2,20 +2,35 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { act } from 'react-dom/test-utils';
 // import chronoscope from 'react-chronoscope';
-import createTree from './CreateTree';
+import createTree from './createTree';
+import TicTacToe from '../mockData/ticTacToe'
 
-function Dialog({ children }) {
-  return (
-    <div>
-      <p>Hello</p>
+  function Dialog({children}){
+    return(
+      <div>
+        <p>Hello</p>
+        {children}
+      </div>
+    )
+  }
+  function Button({children}){
+    return (
+      <button>
```

Comment:
can we also remove the Dialog, Button, App?

Reply:
Sure